### PR TITLE
Add test for sorted_dict function present in export_helpers helper file.

### LINKED ISSUE #5320: Improve the testability of the project by adding unit tests
**Is your feature request related to a problem? Please describe.**

Right now the code coverage is very low and we should improve it by adding unit tests wherever possible.

### PR DESCRIPTION
**Pull request by Anupam-dagar**
Reference: #5320

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The unit tests pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

- [x] All the functions created/modified in this PR contain relevant docstrings.

#### Short description of what this resolves:
This adds test for `sorted_dict` function present in `exported_helpers` file.

#### Changes proposed in this pull request:

- Add test for `sorted_dict`

### CODE REVIEW

**Comment by iamareebjamal** [comment]
This is a unit test, not an integration

**Comment by Anupam-dagar** [comment]
@iamareebjamal I have updated my PR with the required changes.

File: tests/all/unit/api/helpers/test_export_helpers.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
+import unittest
+
+from app import current_app as app
+from tests.all.integration.utils import OpenEventTestCase
+from tests.all.integration.setup_database import Setup
```
Comment (iamareebjamal) [inline tests/all/unit/api/helpers/test_export_helpers.py]: This is unit test package, and you are importing integation test utils and setup_database. You have not written a unit test here. It's still an integration test

Comment (iamareebjamal) [reply tests/all/unit/api/helpers/test_export_helpers.py]: Think yourself, why do I need to setup app context and DB to test ordering of list an dictionary?

Comment (Anupam-dagar) [reply tests/all/unit/api/helpers/test_export_helpers.py]: I guess the test is now correct, please review.

**Pull request merged**